test/relayFeeCalculator.test.ts

```diff
@@ -36,12 +36,12 @@ const testCapitalCostsConfig: { [token: string]: any } = {
 // Example of how to write this query class
 class ExampleQueries implements QueryInterface {
   constructor(private defaultGas = "305572") {}
-  async getGasCosts(): Promise<BigNumberish> {
-    return gasCost(this.defaultGas, 1e9); // 1 gwei
+  getGasCosts(): Promise<BigNumberish> {
+    return Promise.resolve(gasCost(this.defaultGas, 1e9)); // 1 gwei
```

Comment:
Why can't you just return `gasCost` if `gasCost` returns a Promise?

Reply:
`gasCost` unfortunately doesn't return a promise:

```
/**
 * Convert a gas amount and gas price to wei.
 *
 * @param {number} gas - gas amount.
 * @param {BigNumberish} gasPrice - gas price in gwei.
 * @returns {BigNumber} - total fees in wei.
 */
export const gasCost = (gas: BigNumberish, gasPrice: BigNumberish): BigNumber => {
  return BigNumber.from(gas).mul(gasPrice);
};
```